src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Resources/views/translation.html.php

```diff
@@ -46,4 +46,4 @@
 
 <?php echo $view['translator']->trans('typecast', ['a' => (int) '123'], 'not_messages'); ?>
 <?php echo $view['translator']->transChoice('msg1', 10 + 1, [], 'not_messages'); ?>
-<?php echo $view['translator']->transChoice('msg2', intval(4.5), [], 'not_messages'); ?>
+<?php echo $view['translator']->transChoice('msg2', (int)4.5, [], 'not_messages'); ?>
```

Comment:
The corresponding test case is processing the PHP token stream. And this one is here to make sure that a function/method call wouldn't break.